index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
```

Comment:
### Optional
- [ ] It would be better if we properly for our PR, let us leave a space between the `-` and the `text`

Reply:
Done, thank you.